sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLQueryPlanner.java

```diff
@@ -187,6 +189,8 @@ private BeamRelNode convertToBeamRelInternal(String sql, QueryParameters queryPa
     return (BeamRelNode) plannerImpl.transform(0, desiredTraits, root.rel);
   }
 
+  // TODO: Resolve later
+  @SuppressWarnings("nullness")
```

Comment:
What should be replace with null for `config.parserFactory(SqlParserImplFactory.class, null);`

Reply:
I'm guessing this is just a Calcite API thing and we have to suppress warnings because we do not have annotated Calcite? So it is OK to suppress but try to get the scope of suppression as local as possible.